index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="style.css" rel="stylesheet" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <div class="hamburger-menu">
+        <div class="bars">
+          <div class="bars1"></div>
+          <div class="bars2"></div>
+          <div class="bars3"></div>
+        </div>
```

Comment:
- [ ] kindly consider adding these hamburger bars in a nav tag so that  it will later contain nav links to your website and this is for semantic reasons